ArduCopter/mode.cpp

```diff
@@ -200,6 +200,20 @@ bool Copter::set_mode(control_mode_t mode, mode_reason_t reason)
     }
 #endif
 
+    // ensure vehicle doesn't leap off the ground if a user switches
```

Comment:
I think we should add #if FRAME_CONFIG != HELI_FRAME around this check

Reply:
Added.